packages/dotcom-ui-layout/browser.js

```diff
@@ -8,8 +8,7 @@ import oTypography from 'o-typography'
 import 'focus-visible'
 
 export function init({ headerOptions = {}, footerOptions = {} } = {}) {
-  const rootElement = document.querySelector('.n-layout')
-  oTypography.init(rootElement)
+  oTypography.init()
```

Comment:
Looks like this was an oversight on my part in https://github.com/Financial-Times/dotcom-page-kit/pull/770

The intention was for base styles package to provide all font loading related code, including this. For whatever reason this must have gotten lost but I believe the layout package should have imported and used the base styles JS instead: https://github.com/Financial-Times/dotcom-page-kit/blob/master/packages/dotcom-ui-base-styles/browser.js